### PR TITLE
FOPTS-3031 Update parameters of Azure Rightsize Managed Disk policy

### PR DESCRIPTION
**Pull request by midir99**
### Description

- Updated the descriptions and labels of the IOPS and throughput parameters in the README and policy template files.
- Updated the short description of the policy.
- Changed the functionality of `param_min_savings`: Before this version, the `param_min_savings` parameter was used to consider the total savings (the sum of all the savings per resource) and not the savings per resource to decide whether to recommend or not. In this new version, this parameter is used to recommend or not based on the savings of each resource, just as other policies do.

### Issues Resolved

- https://flexera.atlassian.net/browse/FOPTS-3170

### Link to Example Applied Policy

You can find the link to the applied policy in the following comment at Jira:
https://flexera.atlassian.net/browse/FOPTS-3170?focusedCommentId=2248041

### Contribution Check List

- [x] New functionality includes testing.
- [X] New functionality has been documented in the README if applicable
- [X] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: cost/azure/rightsize_managed_disks/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v1.2
+
+- Updated the descriptions and labels of the IOPS and throughput parameters in the README and policy template files.
+- Updated the short description of the policy.
+- Changed the functionality of `param_min_savings`: Before this version, the `param_min_savings` parameter was used to consider the total savings (the sum of all the savings per resource) and not the savings per resource to decide whether to recommend or not. In this new version, this parameter is used to recommend or not based on the savings of each resource, just as other policies do.
```
Comment (TDubovchenko) [inline cost/azure/rightsize_managed_disks/CHANGELOG.md:7]: Perhaps: "Fixed  the functionality of Minimum Savings Threshold parameter. This parameter is used to suppress recommendations with potential savings below specified threshold.

File: cost/azure/rightsize_managed_disks/azure_rightsize_managed_disks.pt [hunk 2]
```diff
@@ -1,13 +1,13 @@
-name "Azure Rightsize Managed Disks"
+name "Azure Rightsize Managed Disks v1.2"
```
Comment (TDubovchenko) [inline cost/azure/rightsize_managed_disks/azure_rightsize_managed_disks.pt]: We usually do not add version to policy name. Should we consider to rename policy to be explicit that it downgrades disk type only.

File: cost/azure/rightsize_managed_disks/CHANGELOG.md (hunk 1 above)
Comment (midir99) [reply cost/azure/rightsize_managed_disks/CHANGELOG.md]: thank you! done!

File: cost/azure/rightsize_managed_disks/azure_rightsize_managed_disks.pt (hunk 2 above)
Comment (midir99) [reply cost/azure/rightsize_managed_disks/azure_rightsize_managed_disks.pt]: thank you! I put it accidentally! I just removed it.

**Review by XOmniverse — APPROVED** [review]
LGTM

**Review by night-codes — APPROVED** [review]
LGTM

**Pull request merged**